admin_manual/configuration_files/big_file_upload_configuration.rst

```diff
@@ -65,7 +65,7 @@ for how to configure those values correctly:
 
 Apache
 ^^^^^^
-* `LimitRequestBody <https://httpd.apache.org/docs/current/en/mod/core.html#limitrequestbody>`_ (In Apache HTTP Server <=2.4.53 this defaulted to unlimited, but now defaults to 1 GiB. The new default limits uploads from non-chunking clients to 1 GiB. If this is a concern in your environment, override the new default by either manually setting it to ``0`` or to a value similar to that used for your local environment's PHP ``upload_max_filesize / post_max_size / memory_limit`` parameters.)
+* `LimitRequestBody <https://httpd.apache.org/docs/current/en/mod/core.html#limitrequestbody>`_ (In Apache HTTP Server <=2.4.53 this defaulted to unlimited, but now defaults to 1 GiB. The new default limits uploads from non-chunking clients to 1 GiB. It also impacts chunked uploads if a maximum chunk size >1 GiB is permitted by the client configuration. If this is a concern in your environment, override the new default by either manually setting it to ``0`` or to a value similar to that used for your local environment's PHP ``upload_max_filesize / post_max_size / memory_limit`` parameters. For chunking clients, as an alternative, the default maximum chunk size can be reduced to 1 GiB. The trade-off is the maximum upload file size (assuming no other limitations elsewhere) will be cut to 10000 TiB rather than 50000 TiB - i.e 10,000 * maxChunkSize.)
```

Comment:
```suggestion
* `LimitRequestBody <https://httpd.apache.org/docs/current/en/mod/core.html#limitrequestbody>`_ (In Apache HTTP Server <=2.4.53 this defaulted to unlimited, but now defaults to 1 GiB. The new default limits uploads from non-chunking clients to 1 GiB and also limits the maximum chunk size for recent versions of the official Nextcloud clients to 1 GiB even though they may require up to 5 GiB. This will break uploads >1 GiB under various circumstances. Our recommendation is to set this Apache parameter to 5 GiB or to the old default behavior of unlimited. Also make sure to review your local environment's PHP ``upload_max_filesize / post_max_size / memory_limit`` parameters to make sure they are at least the same size. For chunking capable clients that support manual configuration, as an alternative, the default maximum chunk size can be reduced to 1 GiB, but each client installation will need this parameter to then be manually configured.)
```